### ISSUE TITLE
ejs include doesn't work

### ISSUE THREAD

**Issue by elie222** [infinitered/gluegun#570]
Been playing around with the package, but ejs `include` doesn't seem to work. 

Error:

```
Could not find the include file "myFile"
    at getIncludePath (app/node_modules/ejs/lib/ejs.js:165:13)
```

**Comment by geovanneoliveira** [comment]
@elie222 I have the same problem!

` Could not find the include file "def.ejs"
    at getIncludePath (/home/geovanne/Documentos/SmartLabs/knot/node_modules/ejs/lib/ejs.js:165:13)`

**Comment by jonatanlins** [comment]
I'm having this problem too

**Comment by jamonholmgren** [comment]
Fixed by #587 and will be released shortly!
